### PR TITLE
Database & Image UI

### PR DESCRIPTION
**Pull request by fearnworks**

### CODE REVIEW

File: backend/fabricateserver/api/image_router.py
Comment (crfearnworks) [inline backend/fabricateserver/api/image_router.py:1]: I like the change to UID

File: webui/src/components/ImageCarousel.vue
Comment (crfearnworks) [inline webui/src/components/ImageCarousel.vue:1]: I'm guessing having a carousel class is more efficient here. It looks like it is. I'll need to add this to a deepdive at some point.

File: webui/tests/unit/ImageCard.test.ts
Comment (crfearnworks) [inline webui/tests/unit/ImageCard.test.ts:1]: Having the IP in here might bite us later? Just something to remember.

**Review by KaminaDuck — APPROVED** [review]
Tested on local system and received doggo image

**Pull request merged**